### DISCUSSION TITLE
How do i install this

### DISCUSSION THREAD

**Discussion by yessir9997** [Joshua-Ashton/VPhysics-Jolt#124] [Q&A]
for some reason when i try to replace vphysics with Vphysics jolt it just comes up with a error when a try to open gmod

**Comment by niceEli** [comment]
you need to set it to chromium x86-64 build and then install to the win64 folder in bin

**Comment by niceEli** [comment]
https://www.youtube.com/watch?v=OdZlsiqynO8
